timemachine/ff/__init__.py

```diff
@@ -84,7 +84,7 @@ def load_from_file(cls, path_or_str: Union[str, Path]) -> "Forcefield":
         path = Path(path_or_str)  # Safe to construct a Path object from another Path object
 
         with resources.files("timemachine.ff.params") as params_path:
-            built_in_path = params_path / original_path
+            built_in_path = params_path / path.name
```

Comment:
Question: would this change allow `/foo/bar/smirnoff_2_0_0_ccc.py` to match a built-in forcefield?

Reply:
Yes, which was the expectation we set at the time. 

We could perhaps change that expectation, though I would probably err on the side of not having multiple ffs that have the same name but different parameters?

Reply:
Ah, I see. I was under the impression that we would not fall back to a builtin FF if an absolute path is passed (even if the basename is the same as a builtin FF). The comment added in https://github.com/proteneer/timemachine/pull/986/commits/0f879aa3aa44445f0453a6a7246cdc151ef09e63 clarifies for me.